src/vs/workbench/api/common/extHostTypes.ts

```diff
@@ -2206,6 +2206,9 @@ export class RelativePattern implements IRelativePattern {
 
 		if (typeof base === 'string') {
 			this.base = base;
+		} else if (URI.isUri(base)) {
+			this.baseFolder = base;
```

Comment:
Why is this a `baseFolder`? Also, what is baseFolder and is it API creep? It's not declared in `vscode.d.ts`